chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/job/CertifiedProductDownloadableResourceCreatorJob.java

```diff
@@ -106,8 +112,8 @@ private void initializeWritingToFiles(final CertifiedProductXmlPresenter xmlPres
         List<CertificationCriterionDTO> criteria = getCriteriaDao().findByCertificationEditionYear(edition)
                 .stream()
                 .filter(cr -> !cr.getRemoved())
-                .sorted((crA, crB) -> Util.sortCriteria(crA, crB))
-                .collect(Collectors.<CertificationCriterionDTO>toList());
+                .sorted((crA, crB) -> criterionService.sortCriteria(crA, crB))
+                .collect(Collectors.<CertificationCriterionDTO> toList());
```

Comment:
I've see these show up before. Is this your auto-formatter doing it? I ask because when I set these lines like this I get a checkstyle error about `'>' is followed by whitespace.`